core/templates/dev/head/components/share/sharing_collection_links_directive.html

```diff
@@ -0,0 +1,74 @@
+<script type="text/ng-template" id="components/sharingCollectionLinks">
```

Comment:
Can we refactor so that the exploration and collection share buttons use the same directive, passing in args as needed for the URL and whether to show the embed button or not?